DigitalLearningSolutions.Data/Services/SectionContentService.cs

```diff
@@ -111,7 +114,6 @@ FROM Tutorials
                             ON Applications.ApplicationID = Customisations.ApplicationID
                         INNER JOIN Sections
                             ON Sections.SectionID = Tutorials.SectionID
-                        CROSS JOIN OtherSectionsExistTable
```

Comment:
Should be able to remove the definition of OtherSectionsExistTable? 🙂

Reply:
Yep, sorry! Removed in 3994927 🙂